src/Entities/Boat.h

```diff
@@ -13,14 +13,26 @@
 
 
 
-
+// tolua_begin
 
 class cBoat :
 	public cEntity
 {
 	typedef cEntity super;
 
 public:
+	enum eType
```

Comment:
Save as `Type` to NBT, but use `Material` everywhere else? I personally prefer code (and API) that is descriptive.